modules/api/src/main/java/net/countercraft/movecraft/craft/Craft.java

```diff
@@ -68,6 +68,7 @@
     private int numMoves;
     @NotNull private final Map<MovecraftLocation,Material> phaseBlocks = new HashMap<>();
     @NotNull private final HashMap<UUID, Location> crewSigns = new HashMap<>();
+    private String uniqueName = "";
```

Comment:
Add a `NotNull` annotation

Reply:
This variable is never guaranteed to be unique. Avoid calling as such to prevent confusion.